gallery.html

```diff
@@ -42,6 +42,14 @@ <h2 id="gallery_heading" class="text-center">Gallery</h2>
         </div>
 
     </section>
+    <style>
```

Comment:
Why are you adding style here if you are adding the same thing in the gallery.css file?

Reply:
by mistake I did this , same thing is done in CSS file